web/crux-ui/src/server/crux/registry-service.ts

```diff
@@ -119,9 +119,16 @@ class DyoRegistryService {
             ...res.gitlab,
             selfManaged: !!res.gitlab.apiUrl,
           }
-        : {
+        : res.github
+        ? {
             type: 'github',
             ...res.github,
+          }       
+        : {
+            type: 'google',
+            ...res.google,
+            token: !!res.google.token ? Buffer.from(res.google.token, 'base64').toString('utf8') : null,
```

Comment:
Could `btoa` work here?

Reply:
As we discussed consistency is more important, we use buffers, in most cases